pages/opam-using.html

```diff
@@ -1,57 +1,74 @@
-<#def TITLE>Installing Coq via OPAM</#def>
+<#def TITLE>Install Coq with opam</#def>
 <#def OCAMLV>4.02.3</#def>
 <#include "incl/macros.html">
 <#include "incl/header.html">
-<h2>What is OPAM</h2>
+<h2>What is opam?</h2>
 
-<p>OPAM is the package manager for the OCaml programming language, the language
+<p>Opam is the package manager for the OCaml programming language, the language
 in which Coq is implemented.
+Opam 2 is the recommended version, and is assumed below.
 Instructions on
-<a href="http://opam.ocaml.org/doc/Install.html">how to install OPAM</a>
-itself are available on the OPAM website.</p>
+<a href="https://opam.ocaml.org/doc/Install.html">how to install opam</a>
+itself are available on the opam website.
+The following command displays the version of opam you have installed:</p>
 
-<p>By following the instructions on this web page one installs the last stable
-version of Coq and all additional packages in the directory <code>~/opam-coq.<#CURRENTVERSION></code>.
-Instructions target an OPAM newcomer.</p>
+<pre><code>opam --version
+</pre></code>
 
-<h2>Using OPAM to install Coq</h2>
+<p>Follow the instructions below to install the last stable version of Coq and
+additional packages. Instructions target an opam newcomer.</p>
 
-<p>Once the <code>opam</code> command is available, i.e. OPAM is installed, one can
-proceed as follows (using opam 2):</p>
+<h2>Initialize opam</h2>
 
-<pre><code>export OPAMROOT=~/opam-coq.<#CURRENTVERSION> # installation directory
-opam init -n --comp=ocaml-base-compiler.<#OCAMLV> -j 2 # 2 is the number of CPU cores
-opam repo add coq-released http://coq.inria.fr/opam/released
-opam install coq.<#CURRENTVERSION> &amp;&amp; opam pin add coq <#CURRENTVERSION>
+<p>Once opam is installed, it must be initialized before its first usage:</p>
+
+<pre><code>opam init
+eval $(opam env)
 </code></pre>
 
-<p>If using <code>opam 1.2</code>, the <code>init</code> command should be:</p>
+<p><code>opam init</code> will prompt you to allow opam to set up
+initialization scripts, which is generally fine to accept. Otherwise, every
+time a new shell is opened you have to type in the
+<code>eval $(opam env)</code> command above to update environment variables.</p>
+
+<p>By default, opam will use the global installation of OCaml. You can initialize
+opam with an explicit compiler version, for example <#OCAMLV>, with the option
+<code>--compiler=ocaml-base-compiler.<#OCAMLV></code>.
+See also the section "Advanced usage" below, about switches and roots.
+</p>
+
+<h2>Install Coq</h2>
 
-<pre><code>opam init -n --comp <#OCAMLV>
+<pre><code># Activate the repository for Coq packages.
+opam repo add coq-released https://coq.inria.fr/opam/released
```

Comment:
This is actually useless to install Coq itself, so it may make sense to postpone this command to the section on Coq packages.